source/tests/system/nirfmxlte_driver_api_tests.cpp

```diff
@@ -54,16 +55,34 @@ InitializeResponse init(const client::StubPtr& stub, const std::string& model)
   return client::initialize(stub, "FakeDevice", options);
 }
 
+nirfsa_grpc::InitWithOptionsResponse init_rfsa(const nirfsa_client::StubPtr& stub, const std::string& resource_name)
+{
+  return nirfsa_client::init_with_options(stub, resource_name, false, false, "Simulate=1, DriverSetup=Model:5663E");
+}
+
 TEST_F(NiRFmxLTEDriverApiTests, Init_Close_Succeeds)
 {
   auto init_response = init(stub(), PXI_5663E);
   auto session = init_response.instrument();
-  // TODO: use EXPECT_SUCCESS here when LTE has get_error and get_error_string
-  EXPECT_RESPONSE_SUCCESS(init_response);
+  EXPECT_SUCCESS(session, init_response);
+
+  auto close_response = client::close(stub(), session, 0);
+
+  EXPECT_SUCCESS(session, close_response);
```

Comment:
The tests for other RFmx flavors use EXPECT_RESPONSE_SUCCESS for the close_response. I'm not sure which is more correct, but we should be consistent.
(Same goes for the test below this one.)

Reply:
I looked at at the Instr ones again and there's a note above EXPECT_RESPONSE_SUCCESS on the close response about "Don't use EXPECT_STATUS because that checks the error text, which can report stale errors from previous tests". So I'll go with that and use EXPECT_RESPONSE_SUCCESS for the two close responses when I post a new PR targeted to main.